concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md

```diff
@@ -1,3 +1,5 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
 #include tutorials-intro-installationnote
 
```

Comment:
Line 9 (below):
`Each section in this tutorial covers a single configuration step.`

See my comment above regarding "covers".

Reply:
Noted